Java/src/main/java/com/gildedrose/BackstagePassItemUpdater.java

```diff
@@ -1,57 +1,38 @@
 package com.gildedrose;
 
-public class BackstagePassItemUpdater extends ItemUpdater {
+class BackstagePassItemUpdater extends ItemUpdater {
+    private static final int EXPIRED_QUALITY_VALUE = 0;
+    private static final int SELL_IN_VALUE_FIVE = 5;
+    private static final int SELL_IN_VALUE_TEN = 10;
```

Comment:
I wonder if there is a point of using the `final` `int`s, as their names include the value eg. `SELL_IN_VALUE_fiive` is `5`. Why did you pick such design?